packs/equipment/highhelm-stronghold-plate.json

```diff
@@ -13,7 +13,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>This <em>+2 resilient fortification bastion plate</em> is made almost entirely of stone plates. It looks almost unwearable and is in fact only wearable because of numerous minor enchantments for comfort and mobility. If you have armor specialization with heavy armor, your resistance while wearing <em>Highhelm stronghold plate</em> applies to both slashing and piercing damage.</p>\n<p><strong>Activate</strong> <span class=\"\\&quot;pf2-icon\\&quot; pf2-icon\">3</span> envision, Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> The plating on your armor explodes off of your body, expands, and reshapes, transforming into a stone structure with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Stone]</em>. While the armor is transformed in this way, you don't gain the benefits of wearing the armor. The armor remains transformed for up to 10 minutes or until you Dismiss the effect, at which point the <em>wall of stone</em> disappears and the armor returns to you, donning itself immediately.</p>\n<hr />\n<p><strong>Crafting Requirements</strong> Supply one casting of <em>wall of stone</em>.</p>"
+            "value": "<p>This <em>+2 resilient fortification bastion plate</em> is made almost entirely of stone plates. It looks almost unwearable and is in fact only wearable because of numerous minor enchantments for comfort and mobility. If you have armor specialization with heavy armor, your resistance while wearing <em>Highhelm stronghold plate</em> applies to both slashing and piercing damage.</p>\n<p><strong>Activate</strong> <span class="pf2-icon">3</span> envision, Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> The plating on your armor explodes off of your body, expands, and reshapes, transforming into a stone structure with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Stone]</em>. While the armor is transformed in this way, you don't gain the benefits of wearing the armor. The armor remains transformed for up to 10 minutes or until you Dismiss the effect, at which point the <em>wall of stone</em> disappears and the armor returns to you, donning itself immediately.</p>\n<hr />\n<p><strong>Crafting Requirements</strong> Supply one casting of <em>wall of stone</em>.</p>"
```

Comment:
```suggestion
            "value": "<p>This <em>+2 resilient fortification bastion plate</em> is made almost entirely of stone plates. It looks almost unwearable and is in fact only wearable because of numerous minor enchantments for comfort and mobility. If you have armor specialization with heavy armor, your resistance while wearing <em>Highhelm stronghold plate</em> applies to both slashing and piercing damage.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">3</span> envision, Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> The plating on your armor explodes off of your body, expands, and reshapes, transforming into a stone structure with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Wall of Stone]</em>. While the armor is transformed in this way, you don't gain the benefits of wearing the armor. The armor remains transformed for up to 10 minutes or until you Dismiss the effect, at which point the <em>wall of stone</em> disappears and the armor returns to you, donning itself immediately.</p>\n<hr />\n<p><strong>Crafting Requirements</strong> Supply one casting of <em>wall of stone</em>.</p>"
```